### PR TITLE
Added GitHub Pull Request Template

### PR DESCRIPTION
**Pull request by tswann**
- Includes code review checklist
- Basic template to provide consistent structure for PRs

Please also review the ODP Code Quality document - this is linked in the Technical Overview of the wiki.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+A checklist is for the benefit of the primary reviewer - aka __YOU__ - the person submitting the pull request.
+
+[] Unit tests written and passing
+[] Comment and coding standards adhered to
+[] Error Handling
+[] Security considered
+[] Performance considered
+[] Appropriate use of logging
+[] README updated?
+
```
Comment (tmrowe) [inline .github/PULL_REQUEST_TEMPLATE:33]: Options to add:
Is well factored?
Is maintainable?
Is documented?

File: .github/PULL_REQUEST_TEMPLATE [hunk 2]
```diff
@@ -0,0 +1,23 @@
+## Description
+
+A few sentences describing the overall context and goals for the PR's commits.
+
```
Comment (tmrowe) [inline .github/PULL_REQUEST_TEMPLATE:4]: High level description

File: .github/PULL_REQUEST_TEMPLATE [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+## Impacted Areas of Application
+
+List high-level components of the application that this PR will affect.
+
+* External web application
+* ...
+
```
Comment (tmrowe) [inline .github/PULL_REQUEST_TEMPLATE:18]: Third heading:
Listing bullets with changes made.

File: .github/PULL_REQUEST_TEMPLATE [hunk 4]
```diff
@@ -0,0 +1,23 @@
+## Description
+
+A few sentences describing the overall context and goals for the PR's commits.
+
+## Impacted Areas of Application
+
```
Comment (tmrowe) [inline .github/PULL_REQUEST_TEMPLATE:6]: Modified Areas

File: .github/PULL_REQUEST_TEMPLATE [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+List high-level components of the application that this PR will affect.
+
+* External web application
+* ...
+
+## Checklist
+
+A checklist is for the benefit of the primary reviewer - aka __YOU__ - the person submitting the pull request.
```
Comment (tmrowe) [inline .github/PULL_REQUEST_TEMPLATE]: This checklist...
A checklist for the...

**Review by tmrowe — CHANGES_REQUESTED** [review]

**Comment by tswann** [comment]
Hi @tmrowe - please see updates applied in latest commit.

Thanks,
Tom

**Review by tmrowe — APPROVED** [review]

**Pull request merged**